Dockerfile

```diff
@@ -45,10 +45,20 @@ RUN make build
 
 # Run image
 FROM quay.io/konveyor/move2kube:${VERSION}
+WORKDIR /move2kube-api
 ENV MOVE2KUBE_PLATFORM="${MOVE2KUBE_PLATFORM}:api-dockerfile"
+
 # Install move2kube-api
 COPY --from=build_base /go/bin/move2kube-api /bin/move2kube-api
+# Copy static files for Move2Kube UI
+COPY ./dist ./dist
```

Comment:
I assume this will move to the move2kube-ui or move2kube-aio image

Reply:
done